lib/libsys/statfs.2

```diff
@@ -232,7 +232,7 @@ The fields in the
 .Vt statfs
 structure have been defined to provide the parameters relevant for
 traditional
-.Tm UNIX
+.Tn UNIX
```

Comment:
Tn is deprecated and doesn't do anything, so I usually remove it and join the lines.